build.go

```diff
@@ -162,21 +162,20 @@ func (c *Client) Build(ctx context.Context, opts BuildOptions) error {
 		FileFilter:         opts.FileFilter,
 	}
 
+	lifecycleVersion := ephemeralBuilder.LifecycleDescriptor().Info.Version
 	// Technically the creator is supported as of platform API version 0.3 (lifecycle version 0.7.0+) but earlier versions
 	// have bugs that make using the creator problematic.
-	lifecycleVersion := ephemeralBuilder.LifecycleDescriptor().Info.Version
 	lifecycleSupportsCreator := !lifecycleVersion.LessThan(semver.MustParse("0.7.4"))
 
 	if lifecycleSupportsCreator && (!opts.Publish || opts.TrustBuilder) {
-		// no need to fetch a lifecycle image, it won't be used
 		lifecycleOpts.UseCreator = true
+		// no need to fetch a lifecycle image, it won't be used
 		return c.lifecycle.Execute(ctx, lifecycleOpts)
 	}
 
-	lifecycleImageSupported := !lifecycleVersion.LessThan(semver.MustParse("0.7.5"))
-	if !lifecycleImageSupported {
-		c.logger.Warnf("Lifecycle %s does not have an associated lifecycle image.", lifecycleVersion.String())
-	} else {
+	lifecycleImageSupported := lifecycleVersion.Equal(builder.VersionMustParse("0.6.1")) || !lifecycleVersion.LessThan(semver.MustParse("0.7.5"))
```

Comment:
Do we want to add 0.6.0?

Reply:
Based on the latest state of known builders and the age of that release, I would think it's unlikely that 0.6.0 is used heavily. A minor lifecycle bump to said builder would be minimal effort for added security.